pkgs/tools/virtualization/cloud-init/default.nix

```diff
@@ -19,10 +20,15 @@ in pythonPackages.buildPythonApplication rec {
       --replace /etc $out/etc \
       --replace /lib/systemd $out/lib/systemd \
       --replace 'self.init_system = ""' 'self.init_system = "systemd"'
+    substituteInPlace cloudinit/sources/DataSourceAltCloud.py \
+      --replace /sbin/modprobe ''${kmod}/bin/modprobe \
+      --replace /sbin/udevadm ''${systemd}/bin/udevadm
```

Comment:
Oops, that's wrong. Nothing broke when I removed those, so doesn't seem to be necessary.